packages/core/components/Puck/index.tsx

```diff
@@ -318,7 +318,7 @@ export function Puck({
                             display: "grid",
                             padding: 16,
                             gridTemplateAreas: '"left middle right"',
-                            gridTemplateColumns: "288px auto 288px",
+                            gridTemplateColumns: "288px auto 344px",
```

Comment:
@Yuddomack thanks for updating, although you missed my other comment:

```suggestion
                            gridTemplateColumns: "344px auto 344px",
```